deploy.doc.sh

```diff
@@ -29,6 +29,40 @@
 # (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
 # SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
+check_versions() {
+  packageVersion=$(ruby -rjson -e "j = JSON.parse(File.read('package.json')); puts j['version']")
+  npmVersion=$(npm view coveo-search-ui version)
+  compare_versions ${packageVersion} ${npmVersion}
+  retval=$?
+  if [ ${retval} == 2 ]; then
+    echo "Package version (${packageVersion}) is lower than current npm version (${npmVersion}). Aborting." >&2
```

Comment:
By exiting with something else that 0, it will fail the build. 

You can simply echo the message that the doc will not be deployed, and exit 0.

Reply:
Since this is not actually going to be an "error" exit, should I remove `>&2` from the echo?